File: src/server/Modules/Organization/Modules.Organization.Core/Entities/Department.cs
using System;
using FluentPOS.Shared.Core.Domain;

namespace FluentPOS.Modules.Organization.Core.Entities
{
    public class Department : BaseEntity
    {
        public string Name { get; set; }

        public bool IsGlobalDepartment { get; set; }

        public string Description { get; set; }

        public long? HeadOfDepartment { get; set; }

        public int Production { get; set; }

        public long PolicyId { get; set; }

        public long? ParentId { get; set; }
    }
}

File: src/client/src/app/modules/admin/org/models/department.ts
export interface Department {
    id: string;
    createaAt: string;
    updatedAt: string;
    organizationId: string;
    branchId: string;
    name: string;
    description: string;
    headOfDepartment: boolean;
    production: number;
    policyId: string;
    parentId?: string;
    parentDept?: string;
}
